sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/VirtualMachine.cs

```diff
@@ -111,121 +245,6 @@ public VirtualMachine(string location, string id = default(string), string name
             CustomInit();
         }
 
-        /// <summary>
-        /// Initializes a new instance of the VirtualMachine class.
```

Comment:
Why do we remove those docs? Is it because of the generation tools, or do we remove them manually?

Reply:
the commented parts are copied over from models/VirtualMachine.cs file, and repeating. I removed them so I can more easily see what constructors are accounted for